RecoCTPPS/Configuration/python/recoCTPPS_cff.py

```diff
@@ -2,7 +2,7 @@
 
 from RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff import *
 from RecoCTPPS.TotemRPLocal.ctppsDiamondLocalReconstruction_cff import *
-from RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteProducer_cfi import ctppsLocalTrackLiteProducer
+from RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteProducer_cff import ctppsLocalTrackLiteProducer
 
```

Comment:
ctppsPixelLocalReconstruction is missing in this recoCTPPS sequence

The reason is that #19909. is not yet merged in 92X: this PR cannot be merged befere that one.